src/Datagrid/Datagrid.php

```diff
@@ -382,4 +348,46 @@ private function isFieldAlreadySorted(FieldDescriptionInterface $fieldDescriptio
         return $values[DatagridInterface::SORT_BY]->getName() === $fieldDescription->getName()
             || $values[DatagridInterface::SORT_BY]->getName() === $fieldDescription->getOption('sortable');
     }
+
+    private function buildForm(): FormInterface
+    {
+        if (null !== $this->form) {
+            return $this->form;
+        }
+
+        foreach ($this->getFilters() as $filter) {
+            [$type, $options] = $filter->getRenderSettings();
+
+            $this->formBuilder->add($filter->getFormName(), $type, $options);
+        }
+
+        $hiddenType = HiddenType::class;
```

Comment:
Is this assignment required? Why shouldn't we use the `::class` pseudo-constant where needed?

Reply:
No, I just copied paste the existent code ; it's fixed now.